bootstrap/pesticide_seed.py

```
@@ -0,0 +1,1243 @@
+    target_names = split_top_level(raw_row.get("SICKNS_HLSCT_NM_WEEDS_NM")) or [
+        "대상정보없음"
+    ]
+    pairs: list[tuple[str, str, str]] = []
+    for crop_name in crop_names:
+        for target_name in target_names:
+            pairs.append((crop_name, target_name, infer_target_kind(target_name)))
+    return pairs
+
+
+def populate_database(
+    session: Session, source_files: list[Path], log_every: int, append_mode: bool
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)